README.rst

```diff
@@ -323,4 +323,4 @@ Examples
 JavaScript dependency
 -----------------------------------------------------------
 
-This Python package bundle the ``vue-vtk-js@3.2.2`` JavaScript library. If you would like us to upgrade it, `please reach out <https://www.kitware.com/trame/>`_.
```

Comment:
can you further update the README to mention that WebXR is supported (just for VtkLocalView ?)